files/jira.rb

```diff
@@ -4,6 +4,24 @@
 
 class Jira < BaseHandler
 
+  def create_issue_loop(client, issue_json)
```

Comment:
I'm just an outsider looking in, but why have a loop here? Could you not just have:

```
rescue Exception => e
      puts "#{e.message} - the error was: #{issue.errors}"
```

I was planning on calling a method here for #31 to send something to the TCP.3030 if there was an exception thrown - am I missing something that needs you to loop?